src/platforms/eglstream-kms/server/platform.cpp

```diff
@@ -248,6 +248,15 @@ mge::DisplayPlatform::DisplayPlatform(
     provider = std::make_shared<InterfaceProvider>(display);
 }
 
+mge::DisplayPlatform::~DisplayPlatform()
+{
+    if (display)
+    {
+        eglTerminate(display);
+        display = nullptr;
+    }
+}
```

Comment:
Did I miss committing this comment?

```suggestion
mge::DisplayPlatform::~DisplayPlatform()
{
    if (display != EGL_NO_DISPLAY)
    {
        eglTerminate(display);
    }
}
```

Reply:
Yup :)